packages/jest-reporters/src/utils.ts

```diff
@@ -17,13 +17,58 @@ const PROGRESS_BAR_WIDTH = 40;
 
 export const printDisplayName = (config: Config.ProjectConfig) => {
   const {displayName} = config;
+  const white = chalk.reset.inverse.white;
+  // handle the case that displayName is not provided
```

Comment:
I'm manually handling this because it seems that `jest-validate` doesn't have the ability to validate values in an object. Ideally this would already be handled by jest-validate. Basically, if any of the values in the displayName option are invalid we default it to white